### DISCUSSION TITLE
There is this line on the side panel after the recent firefox update

### DISCUSSION THREAD

**Discussion by MrHeadsh0t1** [mbnuqw/sidebery#1853] [Q&A] [answered]
![Capture](https://github.com/user-attachments/assets/ffccb510-f5fc-47bc-9039-855e8f870742)

any ideas how to fix it?

**Comment by MrHeadsh0t1** [comment, marked as answer]
I found the solution. I just changed the z-index on my userChrome.css to 10. I am leaving this here in case if anyone has the same problem.

**Reply by Ser4ph4** [reply]
Thanks for this ❤️
